website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Heterogeneous Cluster Support
+linkTitle: Heterogeneous Cluster Support
+weight: 80
+---
+
+An operator deployed to a heterogeneous cluster may not support all of the node architectures in the cluster. This has scheduling implications for both the operator as well as its operands.
```

Comment:
WDYT about we change it here for something like: 

This documentation aims to describe how you can develop Operators which can provide support for multiple platforms. It will help you out ensure that you provide a solution that can work on and can be distributed to be used for multiple platforms. In this way, that provides `"Heterogeneous Cluster Support"`.  

Overview
Any recommendations or best practices suggested by the Kubernetes community, such as how to [develop Operator pattern solutions](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) or how to [use controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) are good recommendations for those who are looking to build operator projects with operator-sdk. Also, see [Operator Best Practices](https://sdk.operatorframework.io/docs/best-practices/best-practices). However, here are some common recommendations.

Reply:
Yeah - so effectively, it would be good to define what heterogeneous cluster support actually means. I'll see if I can make that more clear in my second pass.

Reply:
I tried to incorporate a similar definition in my newest draft.